src/server/req_delete.c

```diff
@@ -325,6 +325,21 @@ issue_delete(job *pjob)
 	(void)issue_to_svr(rmt_server, preq, release_req);
 }
 
+/**
+ * @Brief
+ *		decrement entity usage for a single un-instantiated subjob
+ *
+ * @param[in]	parent - pointer to parent Job structure.
+ */
+static void
+decr_single_subjob_usage(job *parent)
+{
+	parent->ji_qs.ji_svrflags &= ~JOB_SVFLG_ArrayJob; /* small hack to decrementing usage for singel un-initialized subjob */
```

Comment:
typo in `single`

Reply:
fixed the typo in commit named rc1 https://github.com/PBSPro/pbspro/commit/b2f22afcadb50bc872cb198d65b51190511aaf45    If fixed the gramo which was not noticed.